### ISSUE TITLE
Phonertc demo works in browser but not working in android

### ISSUE THREAD

**Issue by jyyblue1987** [alongubkin/phonertc#193]
I have follow to http://stackoverflow.com/questions/26410915/phonertc-demo-application.
 so android is working well.

then I have run 
    cordova platform add browser --usegit
    grunt build --force
    cordova run browser

and login view is displayed. I can success login. and call success.
but browser console out "but vendor.js:4 TypeError: Cannot read property 'receiveMessage' of undefined"

then I can not connect with other browser

**Comment by RAGsoft** [comment]
Yeah I'm getting this too. I see this error when using chrome however when I use FF i see a whole bunch of different errors. The demo and my custom implementation works perfectly fine on both Android and iOS which I'm extremely happy about however, having the browser version working would be the icing on the cake.

**Comment by gerson529** [comment]
I'm getting this too.+1

**Comment by gerson529** [comment]
I'm getting this too.
